bentoml/_internal/configuration/__init__.py

```diff
@@ -150,10 +144,3 @@ def load_global_config(bentoml_config_file: t.Optional[str] = None):
     )
 
     BentoMLContainer.config.set(bentoml_configuration.as_dict())
-
-
-def save_global_config(config_file_handle: t.IO[t.Any]):
-    from ..configuration.containers import BentoMLContainer
-
-    content = yaml.safe_dump(BentoMLContainer.config)
-    config_file_handle.write(content)
```

Comment:
I believe we probably don't want to expose this to end user, hence the removal.

Reply:
Why not?

Reply:
We can have this back once we have config versioning.

Reply:
Why are you removing this? Is this even related to this PR?

Reply:
not really, let me revert all the changes in here.